patches/radare2-flatpak-folders.patch

```diff
@@ -0,0 +1,21 @@
+Temporally hardcode flatpak paths for org.radare.iaito
+until radare2 releases a new version with this PR:
+https://github.com/radareorg/radare2/pull/20828
+
+diff --git a/libr/include/r_userconf.h.acr b/libr/include/r_userconf.h.acr
+index fc5edb903a..9413935d5b 100644
+--- a/libr/include/r_userconf.h.acr
++++ b/libr/include/r_userconf.h.acr
+@@ -78,9 +78,9 @@ extern "C" {
+ #define R2_SDB_MAGIC        R_JOIN_2_PATHS (R2_SDB, "magic")
+ #define R2_SDB_FORMAT       R_JOIN_2_PATHS (R2_SDB, "format")
+
+-#define R2_HOME_CONFIGDIR   R_JOIN_2_PATHS (".config", "radare2")
+-#define R2_HOME_DATADIR     R_JOIN_3_PATHS (".local", "share", "radare2")
+-#define R2_HOME_CACHEDIR    R_JOIN_2_PATHS (".cache", "radare2")
++#define R2_HOME_CONFIGDIR   ".var/app/org.radare.iaito/config/radare2"
++#define R2_HOME_DATADIR     ".var/app/org.radare.iaito/data/radare2"
++#define R2_HOME_CACHEDIR    ".var/app/org.radare.iaito/cache/radare2"
```

Comment:
This component is not using QT like the main UI app.

This has been fixed in a [PR](https://github.com/radareorg/radare2/pull/20828/commits), but the next version of this software will be released in 2 months. So this is a temporal workaround until then, and we are aware this will not compile when using the next version of this component.